src/AppBundle/Resources/views/Export/cycleorder.txt.twig

```diff
@@ -0,0 +1,25 @@
+{{ deck.name }} {{ deck.version }}
+
+{{ deck.faction.name }}
+{% for agenda in deck.agendas %}
+{{ agenda.card.name }}
+{% endfor %}
+
+{% set first_pack = deck.included_packs|first %}
+{% set last_pack = deck.included_packs|last %}
+Packs: {% if deck.included_packs|length > 1 %}
+From {{ first_pack.pack.name }}{% if first_pack.nb > 1 %} ({{ first_pack.nb }}){% endif %} to {{ last_pack.pack.name }}{% if last_pack.nb > 1 %} ({{ last_pack.nb }}){% endif %}
+{% else %}
+{{ first_pack.pack.name }}{% if first_pack.nb > 1 %} ({{ first_pack.nb }}){% endif %}
+{% endif %}
+
+{% set slotsByCycle = deck.slots_by_cycle_order %}
+{% for cycle, slots in slotsByCycle  %}
+
+---- {{ cycle }} ----
+{% for slot in slots %}
+#{{ "%03d" | format(slot.card.position) }} {{ slot.quantity }}x {{ slot.card.name }}
```

Comment:
please keep the output format of actual cards consistent with `plain.txt.twig`.

should be 

```html
{{ slot.quantity }}x {{ slot.card.name }} ({{ slot.card.pack.name }})
```

this keeps downloaded lists to re-importable (into thronesdb as well as theironthrone).

Reply:
Hi @stopfstedt! 

Thanks for taking the time to review, I'll start working on these ASAP. I hope I didn't make the job too hard :smile: 

For this specific change, however, is there any way I can keep it as-is? I understand the point of keeping it consistent with the current export format, however the main goal of having the deck exported in this way is to have a fast reference to the cards when building the deck in real life - and it doesn't want to replace the current way the deck is being exported. I believe that card position is a huge help when doing so - I built several decks in this way and card position was possibly the second most important information. 

I admit that I'm really new to the game, so I may find this more helpful than seasoned players, however if it is possible I'd like to discuss some possible alternative to keep the output format in this way - for example by marking it as non-importable. I could also work on the import function so that this format can be imported, if needed.

Reply:
I edited the import function in order to work with the ordered text export - at least in thronesdb. It was commited in 1b9065ea4c831cfdf0ae6903a11285bf5058ef00.

Let me know if this could work.